src/internal/db/__init__.py

```
@@ -1,17 +1,112 @@
+    get_async_engine,
+    get_async_session_factory,
     get_engine,
-    get_session,
-    session_scope,
 )
 
+# ---------------------------------------------------------------------------
+# Public async sessionmaker — lazily initialised on first call
+# ---------------------------------------------------------------------------
+
+def SessionLocal() -> async_sessionmaker[AsyncSession]:
```

Pylint (4.0.5)

[error] Undefined variable 'async_sessionmaker'

(E0602)